arches/app/utils/response.py

```diff
@@ -49,7 +49,9 @@ def __init__(self, content=b"", *args, **kwargs):
 
 
 class JSONErrorResponse(JSONResponse):
-    def __init__(self, title=None, message=None, content={}, status=500, *args, **kwargs):
+    def __init__(self, title=None, message=None, content=None, status=500, *args, **kwargs):
+        if not content:
+            content = {}

```

Comment:
New workflow tests passed in isolation but failed when combined with the `views.resource_tests.CommandLineTests`, traced it to this mutable default argument.

There's a possibility that JSONErrorRepsonses are being generated always with the first delete message rather than the correct message (like the failing test).

Not sure if this is important enough to backport. (I guess it would affect what surfaces in the UI.) If so, I can open a PR.